src/test/java/picard/sam/FilterReadsByFlowCellLocationTest.java

```diff
@@ -0,0 +1,163 @@
+package picard.sam;
+
+import htsjdk.samtools.*;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Unit tests for FilterFlowCellEdgeReads using TestNG.
+ */
+public class FilterReadsByFlowCellLocationTest {
+
+    // Temporary files for input and output.
+    private File inputSam;
```

Comment:
why share these fields across tests?  You're just regenerating them in each test